app.js

```diff
@@ -77,4 +81,20 @@ async function checkSentMessages() {
   }
 }
 
+async function getIncomingMessagesNumber() {
```

Comment:
I already pointed this out a little later, but get "incoming" and"outgoing" messages could be abstracted into one function.